### PR TITLE
add: disabled Testnet BTC API warning

### PR DESCRIPTION
**Pull request by jessiemongeon1**
Thank you for your contribution to the IC Developer Portal. This repo contains the content for https://internetcomputer.org and the ICP Developer Documentation, https://internetcomputer.org/docs/. 

If you are submitting a Pull Request for adding or changing content on the ICP Developer Documentation, please make sure that your contribution meets the following requirements:

- [ ] Follows the [developer docs style guide](style-guide.md).
- [ ] Follows the [best practices and guidelines](README.md#best-practices).
- [ ] New documentation pages include [document tags](README.md#document-tags).
- [ ] New documentation pages include [SEO keywords](README#seo-keywords).
- [ ] New documents are in the `.mdx` file format to support the previous two components. 
- [ ] New documents are registered in [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js), otherwise, it will not appear in the
  side navigation bar.
- [ ] Make sure that the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file is
  filled with new documents that you added. This way we can ensure that future Pull Requests are reviewed by the right people.

### CODE REVIEW

**Review by marc0olo — APPROVED** [review]
@jessiemongeon1 I would replace "in the future" with "soon". maybe we can also add a link to the forum discussion.

**Comment by jessiemongeon1** [comment]
@marc0olo I think "soon" is too relative. I'll add a link to the forum.

File: docs/developer-docs/multi-chain/bitcoin/using-btc/btc-dev-workflow.mdx [hunk 1]
```diff
@@ -21,6 +21,12 @@ To interact with the Bitcoin testnet or mainnet networks, a developer needs to u
 
 1. [Generate a Bitcoin address](generate-addresses.mdx): To receive BTC, your canister must generate a Bitcoin address using the `ecdsa_public_key` API endpoint of the [management canister](/docs/current/developer-docs/smart-contracts/advanced-features/management-canister#signing-and-submitting-bitcoin-transactions).
 
+:::danger
+
+The Bitcoin Testnet API is currently disabled and the ckTestBTC minter canister has been stopped. This is temporary and they will be re-enabled in the future.
```
Comment (jessiemongeon1) [inline docs/developer-docs/multi-chain/bitcoin/using-btc/btc-dev-workflow.mdx:26]:
```suggestion
The Bitcoin Testnet API is currently disabled and the ckTestBTC minter canister has been stopped. This is temporary and they will be re-enabled in the future.

[Learn more](https://forum.dfinity.org/t/direct-integration-with-bitcoin/6147/745).
```

File: docs/developer-docs/multi-chain/bitcoin/using-btc/local-development.mdx [hunk 2]
```diff
@@ -19,17 +19,24 @@ locally. Testing locally allows you to iterate and improve your dapp more quickl
 the [Bitcoin API](/docs/current/references/ic-interface-spec#ic-bitcoin-api)
 so that you can locally test out your dapp before deploying it to the Internet Computer.
 
+:::danger
+
+The Bitcoin Testnet API is currently disabled and the ckTestBTC minter canister has been stopped. This is temporary and they will be re-enabled in the future.
```
Comment (jessiemongeon1) [inline docs/developer-docs/multi-chain/bitcoin/using-btc/local-development.mdx:24]:
```suggestion
The Bitcoin Testnet API is currently disabled and the ckTestBTC minter canister has been stopped. This is temporary and they will be re-enabled in the future.

[Learn more](https://forum.dfinity.org/t/direct-integration-with-bitcoin/6147/745).
```

File: docs/developer-docs/multi-chain/bitcoin/using-btc/submit-transactions.mdx [hunk 3]
```diff
@@ -20,6 +20,12 @@ To submit transactions to the Bitcoin network, the Bitcoin integration API expos
 
 The following snippet shows how to send a signed transaction to the Bitcoin network.
 
+:::danger
+
+The Bitcoin Testnet API is currently disabled and the ckTestBTC minter canister has been stopped. This is temporary and they will be re-enabled in the future.
```
Comment (jessiemongeon1) [inline docs/developer-docs/multi-chain/bitcoin/using-btc/submit-transactions.mdx:25]:
```suggestion
The Bitcoin Testnet API is currently disabled and the ckTestBTC minter canister has been stopped. This is temporary and they will be re-enabled in the future.

[Learn more](https://forum.dfinity.org/t/direct-integration-with-bitcoin/6147/745).
```

**Comment by marc0olo** [comment]
> @marc0olo I think "soon" is too relative. I'll add a link to the forum.

@jessiemongeon1 typically I agree, but we know it is ~2 weeks to arrive. "in the future" implies to me that it might take a long time. maybe "in the near future" is the way to go 🤔 

but anyway, I am fine with everything 😁 👍

**Pull request merged**